sunpy/io/special/asdf/resources/schemas/generic_map-1.0.0.yaml

```diff
@@ -1,7 +1,7 @@
 %YAML 1.1
 ---
 $schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
-id: "asdf://sunpy.org/schemas/generic_map-1.0.0"
+id: "asdf://sunpy.org/sunpy/schemas/generic_map-1.0.0"
```

Comment:
I assume changing this on the old schemas can't cause a problem? There's no way for external things to be referencing it?

Reply:
I don't foresee this causing any issues.

The `id` here is effectively unused by asdf at the moment. asdf will instead lookup schemas based on the uri used to register them (based on the path and the prefix provided to the `DirectoryResourceMapping`). However, if this schema contained a local `$ref` it's possible that `jsonschema` would have issues as it likely uses the `id` in the schema to resolve the ref.